image/metadata/ncsa.image.metadata.py

```diff
@@ -37,17 +38,21 @@ def process_message(self, connector, host, secret_key, resource, parameters):
         result = self.parse_exif(subprocess.check_output(
             [self.args.image_binary, "-verbose", inputfile], stderr=subprocess.STDOUT).decode("utf-8"))
 
-        metadata = {
-            "@context": {
-                "@vocab": "http://www.w3.org/2003/12/exif/ns"
-            },
-            "file_id": file_id,
-            "content": result,
-            "agent": {
-                "@type": "cat:extractor",
-                "extractor_id": host + "/api/extractors/ncsa.image.metadata"
+        if float(clowder_version) >= 2.0:
```

Comment:
Fixed for both pull requests. You are right - the method will work either way and then there's no need to check clowder version here, it'll be done on the pyclowder side.

Reply:
For the metadata and v1, this branch of pyclowder:

https://github.com/clowder-framework/pyclowder/pull/60

Allows the context to be passed to the get_metadata so that the method can be used for both instead of this call.